More_types/23Maps.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+// picパッケージ、stringsパッケージを取得
+import (
+	"strings"
+
+	"golang.org/x/tour/wc"
+)
+
+/**
+* WordCount関数を定義
+* 引数はstring型のsとし、返り値はキーはstring型、値はint型のmapを返す。
+ */
+func WordCount(s string) map[string]int {
+
+	/**
+	 * strings.Fields関数により、引数で渡された文字列を空白文字ごとに分割し、スライスとして返す
+	 * 空白のみが含まれる場合は空のスライスを返す
+	 */
+	word_list := strings.Fields(s)
```

Comment:
ご参考までに。
複雑なロジックとかだと、こんな感じで実例書いちゃうことが多いです。
```suggestion
	word_list := strings.Fields(s) // ex) "  a b cc" -> []string{"a", "b", "cc"}
```

Reply:
ありがとうございます。参考にします！